base/loading.jl

```diff
@@ -90,44 +90,15 @@ function try_path(prefix::String, base::String, name::String)
     return nothing
 end
 
-# `wd` is a working directory to search. defaults to current working directory.
-# if `wd === nothing`, no extra path is searched.
-function find_in_path(name::String, wd)
-    isabspath(name) && return name
-    base = name
-    if endswith(name,".jl")
-        base = name[1:end-3]
-    else
-        name = string(base,".jl")
-    end
-    if wd !== nothing
-        isfile_casesensitive(joinpath(wd,name)) && return joinpath(wd,name)
-    end
-    p = try_path(Pkg.dir(), base, name)
-    p !== nothing && return p
-    for prefix in LOAD_PATH
+function find_in_path(base::String)
+    name = "$base.jl"
+    for prefix in [Pkg.dir(); LOAD_PATH]
```

Comment:
ref https://github.com/JuliaLang/julia/pull/15934#r60162030, this concatenation may still be expensive?